charts/osbuilder/templates/deployment.yaml

```diff
@@ -50,6 +50,10 @@ spec:
                   - --leader-elect
                   - --tool-image={{ .Values.image.repository | default "quay.io/kairos/osbuilder-tools" }}:{{ .Values.image.tag | default .Chart.AppVersion }}
                   - --serve-image={{ .Values.image.serving | default "nginx" }}
+                  - --copy-to-namespace={{ .Release.Namespace }}
+                  - --copy-role=osartifactbuilder-operator-artifactCopier
```

Comment:
ditto, those opts should be all configurable

Reply:
agreed here too